crates/uv-cli/src/lib.rs

```diff
@@ -3824,9 +3824,10 @@ pub struct ToolUninstallArgs {
 #[derive(Args)]
 #[allow(clippy::struct_excessive_bools)]
 pub struct ToolUpgradeArgs {
-    /// The name of the tool to upgrade.
+    /// The name of the tool to upgrade, along with an optional version specifier to constrain
+    /// the upgrade (e.g., `ruff` or `ruff<0.8.0`).
```

Comment:
I would try to avoid a long message here. The first line needs to be brief for the CLI help menu, it shouldn't take multiple lines.